client/my-sites/stats/summary/index.jsx

```diff
@@ -369,7 +358,28 @@ class StatsSummary extends Component {
 				<NavigationHeader className="stats-summary-view" navigationItems={ navigationItems } />
 
 				<div id="my-stats-content" className="stats-summary-view stats-summary__positioned">
-					{ summaryViews }
+					<StatsGlobalValuesContext.Consumer>
```

Comment:
Could we use the conditional operator to simplify conditions to avoid conflicts?

```suggestion
					{ this.props.context.params.module === 'utm' ? (
						<StatsGlobalValuesContext.Consumer>
							{ ( isInternal ) => (
								<>
									{ supportsUTMStats || isInternal ? (
										<>
											{ this.renderSummaryHeader( path, statType, false, moduleQuery ) }
											<StatsModuleUTM
												siteId={ siteId }
												period={ this.props.period }
												query={ moduleQuery }
												summary
											/>
										</>
									) : (
										<div>{ translate( 'This path is not available.' ) }</div>
									) }
								</>
							) }
						</StatsGlobalValuesContext.Consumer>
					) : (
						summaryViews
					) }
```

Reply:
Yes, good point.